Steer/DigitizerWorkflow/src/MCHDigitizerSpec.cxx

```diff
@@ -79,12 +80,18 @@ class MCHDPLDigitizerTask : public o2::base::BaseDPLDigitizer
     auto context = pc.inputs().get<o2::steer::DigitizationContext*>("collisioncontext");
     context->initSimChains(o2::detectors::DetID::MCH, mSimChains);
     const auto& eventRecords = context->getEventRecords();
+    auto timeOffset = DigitFilterParam::Instance().timeOffset;
```

Comment:
just to be sure there is no problem with the correction: usually, we correct InteractionRecord by subtracting a positive delay, in the DigitFilteringSpec you do the opposite: pushing IR to the future. Is this what you want to do?

Reply:
Hi @shahor02 , Yes, this delay has been tuned using pp data and, for some reason, we have to add a positive delay to synchronise MCH with MID, which has been synchronised to the interaction BC.

Reply:
@pillot this is strange but ok, it is what it is, merging.